www/viewgame

```diff
@@ -1479,14 +1484,15 @@ function updateUnwishList(id, stat)
 
 // When the user clicks the "Submit Time" button, get the input 
 // from the hours and minutes fields, validate it, convert it to 
-// minutes only, and save it to the database. 
+// minutes only, and save it to the database (with the note, if present).
 function submitTime() {
     var new_hours = document.getElementById('hours').value;
     var new_minutes = document.getElementById('minutes').value;
+    var time_note = document.getElementById('timeNote').value;
```

Comment:
Should I change this

`var time_note = document.getElementById('timeNote').value;`

to this?

`var time_note = document.getElementById('timeNote').value.trim();`

Reply:
Sure :)

Reply:
But it's good to trim stuff also in the server side before saving.

Reply:
I wasn't sure how server-side trimming would work if encodeURIComponent() is used on line 1525. Will spaces still be trimmable after they've been converted to `%20`?

Reply:
The server automatically unencodes URI components when you access them via `get_req_data()`. So if you type `Without hints` in the UI and it gets converted to `Without%20hints` for transmission to the server, `$timenote` will be set to `Without hints`, so trimming will work normally.